Application/Payments/Queries/GetOrderPaymentQuery.cs

```diff
@@ -6,8 +6,15 @@
 
 namespace Application.Payments;
 
-public record GetOrderPaymentQuery (int orderId) : IRequest<IEnumerable<PaymentDto>>;
-
+public record GetOrderPaymentQuery(int orderId) : IAuthorizedRequest<IEnumerable<PaymentDto>>
+{
+    internal Employee employee;
```

Comment:
fieldai prasideda is didziosios raides

Reply:
internal cia nereikia naudot. nera prasmes saugot employee, jog nebutu pasiekiamas outside Application assembly. palik public Employee Employee { get; init; }